pyInterface/rootPwaPy.cc

```diff
@@ -101,6 +102,7 @@ BOOST_PYTHON_MODULE(libRootPwaPy){
 	rpwa::py::exportCalcAmplitude();
 	rpwa::py::exportPwaLikelihood();
 	rpwa::py::exportPwaFit();
+
 #ifdef USE_NLOPT
```

Comment:
Could you please remove these newlines, this file does not have to be changed at all.